soc/arm/nxp_imx/rt/arm_mpu_regions.c

```diff
@@ -20,7 +20,7 @@ static const struct arm_mpu_region mpu_regions[] = {
 	/* Region 1 */
 	MPU_REGION_ENTRY("SRAM_0",
 			 CONFIG_SRAM_BASE_ADDRESS,
-			 REGION_RAM_ATTR(REGION_SRAM_0_SIZE)),
+			 {(NORMAL_OUTER_INNER_WRITE_BACK_WRITE_READ_ALLOCATE_NON_SHAREABLE) | (REGION_SRAM_0_SIZE) | (P_RW_U_RW_Msk)}),
```

Comment:
I'm not sure we really want to make all of the RAM read/write. Maybe we can add a new MPU region instead?